packages/app-builder/.env.example

```diff
@@ -10,7 +10,8 @@ NODE_ENV=development
 SESSION_SECRET=SESSION_SECRET
 SESSION_MAX_AGE=43200
 
-MARBLE_API_DOMAIN=http://localhost:8080
+MARBLE_API_DOMAIN_CLIENT=http://localhost:8080
```

Comment:
The one case where the two values are not equivalent is if the application (api+front) are running locally in docker containers. In this case, the client (browser) can call the API on localhost but the front server cannot